public-images/public-images-reference/artifacts.rst

```diff
@@ -0,0 +1,1495 @@
+
+QEMU Copy On Write (QCOW)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.img``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64.img``
+   * - Format description
+     - QCOW image files are disk image files containing raw sector-by-sector copies of a storage device.
+   * - Use cases
+     - QCOW images are used for creating bootable disks and virtual machines in virtualisation environments. Some of the primary features that make QCOW images attractive are their support for dynamic disk sizing, snapshot support and copy-on-write. You can use a ``.img`` file to create a bootable USB drive or a virtual machine disk in QEMU or VirtualBox.
+
+Root tarball
+~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>-root.tar.xz``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64-root.tar.xz``
+   * - Format description
+     - Tar archive compressed with XZ (LZMA2) containing a root file system for various architectures.
+   * - Use cases
+     - These files are used for deploying base system images in virtual machines and containers. You can use ``.root.tar.xz`` files to distribute pre-configured root file systems that can be deployed directly into virtual machines or container runtimes like Docker or Kubernetes.
+
+SquashFS
+~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.squashfs``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64.squashfs``
+   * - Format description
+     - SquashFS is a compressed read-only file system format.
+   * - Use cases
+     - SquashFS files are used for embedding file systems in read-only environments, often in embedded systems or live CDs. In cloud environments, they are used for distributing lightweight operating system images that are ready to use. You can use a ``.squashfs`` file containing a minimal Linux distribution to create container images that boot quickly and require minimal storage space.
+
+Tarball (gzip)
+~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>.tar.gz``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64.tar.gz``
+   * - Format description
+     - ``.tar.gz`` is an archive format, often called a tarball, that combines files into a single file and compresses it using ``gzip`` compression.
+   * - Use cases
+     - Our ``.tar.gz`` archives are used to distribute complete file system images along with the kernel for various operating systems and virtualisation platforms. They allow extraction and booting of the entire system on compatible hardware or virtual machines.
+
+Vagrant Box
```

Comment:
```suggestion
Vagrant box
```
[Hashicorp](https://developer.hashicorp.com/vagrant/docs/boxes) doesn't seem to capitalise 'box' and we haven't done it in our how-to guide about it too. So it would be good to maintain consistency.

Reply:
You are correct after I double checked those locations as well. I think their capitalization in the title headings is what threw me. Forgetting that not everyone is using sentence-case for headings.